### PR TITLE
Run test cases prefixed with t_ with CommonTest runner

### PR DESCRIPTION
**Pull request by dmitrivereshchagin**
Also simplify test pattern customization for CommonTest runner.

### CODE REVIEW

File: autoload/test/erlang.vim [hunk 1]
```diff
@@ -1,4 +0,0 @@
-let test#erlang#patterns = {
-  \ 'test':      ['\v^\s*(test_\w*)\('],
-  \ 'namespace': [],
-\}
```
Comment (codeinabox) [inline autoload/test/erlang.vim:4]: This shouldn't need to be deleted

Comment (dmitrivereshchagin) [reply autoload/test/erlang.vim:4]: I think it's OK to delete it.

This variable name is misleading.  The variable isn't used (and cannot be) in all Erlang runners.  It also requires to specify namespace key that cannot be used and is simply ignored.  And it's not documented.

I also noticed that similar variables are often defined unconditionally along with functions in the same autoload script.  That is, in such a way that they are difficult to override.

Is there any reason other than minor BC break to keep this variable?

Comment (codeinabox) [reply autoload/test/erlang.vim:4]: Alright, you've convinced me, and I've noticed that not every file type has or needs this

**Review by codeinabox — APPROVED** [review]
Thank you!

**Pull request merged**